xml/cap_depl_azure.xml

```diff
@@ -235,11 +235,17 @@ cap-aks</screen>
    List the &kube; versions currently supported by AKS (see
    <link xlink:href="https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions"/>
    for more information on the AKS version support policy). When creating your
-   cluster in the next step, specify a &kube; version for consistent
-   deployments:
+   cluster in the next step, specify the newest supported &kube; version:
```

Comment:
I don't think we should instruct users to always use the newest version as the available versions are updated quite frequently and the newest version may not have been tested by our team yet.

Reply:
OK